connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorCheckpointConnector.java

```diff
@@ -71,9 +72,9 @@ public void start(Map<String, String> props) {
         String connectorName = config.connectorName();
         sourceAndTarget = new SourceAndTarget(config.sourceClusterAlias(), config.targetClusterAlias());
         groupFilter = config.groupFilter();
-        sourceAdminClient = config.forwardingAdmin(config.sourceAdminConfig());
-        targetAdminClient = config.forwardingAdmin(config.targetAdminConfig());
-        scheduler = new Scheduler(MirrorCheckpointConnector.class, config.adminTimeout());
+        sourceAdminClient = config.forwardingAdmin(config.sourceAdminConfig("source-admin"));
+        targetAdminClient = config.forwardingAdmin(config.targetAdminConfig("target-admin"));
+        scheduler = new Scheduler(config.entityLabel(), config.adminTimeout());
```

Comment:
Doesn't this drop information? If the user isn't running MM2 in dedicated mode, the name of the connector may be different than its class. We might want to preserve the class name in these messages.

Reply:
I misunderstood, and thought that this applies to both client.ids and thread names
For the Scheduler name, I will restore the classname part